.github/workflows/automation.yml

```diff
@@ -0,0 +1,61 @@
+name: Automation
+
+on:
+  pull_request:
+  issues:
+  issue_comment:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Remove from 'Current Release' project
+        uses: alex-page/github-project-automation-plus@v0.5.1
+        if: (github.event.pull_request || github.event.issue.pull_request) && !steps.checkLabel.outputs.hasLabel
+        continue-on-error: true
+        with:
+          project: Current Release
+          column: In progress
+          repo-token: ${{ secrets.GH_TOKEN }}
+          action: delete
+
+      - name: Add to 'Release Next' project
+        uses: alex-page/github-project-automation-plus@v0.5.1
+        if: (github.event.pull_request || github.event.issue.pull_request) && github.event.action == 'opened'
+        continue-on-error: true
+        with:
+          project: Release Next
+          column: In progress
+          repo-token: ${{ secrets.GH_TOKEN }}
+
+      - name: Add to 'Current Release' project
+        uses: alex-page/github-project-automation-plus@v0.5.1
+        if: (github.event.pull_request || github.event.issue.pull_request) && steps.checkLabel.outputs.hasLabel
```

Comment:
Does this get triggered when any label is attached?

Reply:
I was more asking how does it know when the “stable-backport” tag is added, vs “feature”  or any other tag

Reply:
@crobibero Ah, I see! Good catch, I missed the step during the rebase. Fixed now!